### PR TITLE
MRG hnn_core integration improvements in code

### PR DESCRIPTION
**Pull request by jasmainak**
Just making some edits as I go through the code @blakecaldwell

### CODE REVIEW

**Comment by jasmainak** [comment]
okay good to go from my end. @cjayb @ntolley @rythorpe do you want to do a pass after this pull request? It will go much faster if we each take some time to iterate over the code and improve it based on what we do in hnn-core

**Comment by jasmainak** [comment]
also @blakecaldwell the code isn't exactly conforming to pep8. I ran autopep8 to fix some of them. Rest has to be fixed iteratively I guess

**Comment by cjayb** [comment]
See #16. No longer crashing, but visualisations not quite there yet.

**Comment by blakecaldwell** [comment]
Thanks @jasmainak!

**Comment by blakecaldwell** [comment]
@jasmainak this needs to be updated after a rebase of https://github.com/jonescompneurolab/hnn/pull/232

BTW, I'm only fixing pep8 for files that I make significant changes to and not doing all of hnn_qt5.py. Would be a good introduction for someone to the GUI

**Comment by jasmainak** [comment]
Rebase done!

File: README.md [hunk 1]
```diff
@@ -25,6 +25,14 @@ potentials (ERPs) and low frequency rhythms (alpha/beta/gamma).
...
 Please follow the links on our [installation page](installer) to find instructions for your operating system.
 
+## Quickstart
+
+Just do:
+
+    $ ./hnn.sh
+
+to start the HNN graphical user interface
+
```
Comment (blakecaldwell) [inline README.md:35]: My opinion is that it'd be better to instruct the user:
```
python hnn.py
```

Python paths were a source of confusion for GUI users. I think making it explicit helps them reason through it.

**Comment by blakecaldwell** [comment]
I like separating the code out in `dialog.py` as you've done. What about naming the file `qt_dialog.py` similar to other files I created (all with code pulled from `hnn_qt5.py`)?

`visvolt.py` hasn't been updated yet (FYI @cjayb). It will get rewritten in the next push to https://github.com/jonescompneurolab/hnn/pull/232

**Comment by jasmainak** [comment]
comments addressed!

File: README.md (hunk 1 above)
Comment (jasmainak) [reply README.md:35]: sounds good to me!

**Comment by hnneurosolver** [comment]
LGTM thanks @jasmainak!

**Pull request merged**